daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/ElementKindParsers.scala

```diff
@@ -21,6 +21,7 @@ import org.apache.daffodil.processors._
 import org.apache.daffodil.util.LogLevel
 import org.apache.daffodil.util.Maybe
 import org.apache.daffodil.exceptions.Assert
+import org.apache.daffodil.util.RangeBound
 
 class ComplexTypeParser(rd: RuntimeData, bodyParser: Parser)
   extends CombinatorParser(rd) {
```

Comment:
Did the next file ".choiceBranchKeyRanges.tdml.swp" get added by mistake?

Reply:
I added this to my global gitignore.

I will also send a separate PR adding this to Daffodil's gitignore.

Reply:
I don't feel strongly about it, but my opinion is that Daffodil's .gitignore shouldn't contain things specific to a particular development environment. It should only contain intermediate files that are part of a build/test. The idea is that we shouldn't have to try to maintain a list of all the different tmp files that different editors might want to throw in our repo. It's up to users to handle that with a global .gitignore based on their dev environment.

That said, for common things like vim and others, it could prevent this kind of issue, so I'm not totally against it. Our gitignore is already kindof a mess with different dev files scattered throughout.

Reply:
We already IntelliJ, Eclipse, and Emacs entries, so I'm in favor of adding Vim on purely religious grounds ;)